doc/docportal/advanced_topics/configuration_file.rst

```diff
@@ -363,6 +364,17 @@ There are many recognized configuration keys. In the table below, each key is ei
 	- 2gs
 	- atari
 	- macintosh "
+		":ref:`retrowaveopl3_bus <adlib>`",string,,"
+	Specifies how the RetroWave OPL3 is connected:
+	- serial (connected to a USB port using a PotatoPi)
```

Comment:
Seems to be an issue with the formatting of the bullet points but to be honest I can’t see why at first glance. I’ll have another look and try to figure it out tomorrow :)